doc/03_reference/verification.rst

```diff
@@ -223,6 +223,138 @@ You can add any compile/runtime options in `dv/uvm/core_ibex/yaml/simulator.yaml
++---------------+--------------------------+------------+--------------------------------------------------------------------------------------------------------------------------------------+
+| Review        | SEC_CM_DV_REVIEWED       | Complete   |                                                                                                                                      |
++---------------+--------------------------+------------+--------------------------------------------------------------------------------------------------------------------------------------+
+
+Ibex SEC_CM Test Mapping
+""""""""""""""""""""""""
+
+The :ref:`security features Ibex implements <security>` are given specific security countermeasure names in OpenTitan (see 'Security Countermeasures' in the `Hardware Interfaces <https://docs.opentitan.org/hw/ip/rv_core_ibex/doc/#hardware-interfaces>`_ documentation section).
+Each countermeasure has a test that exercises it.
+The mapping between countermeasures and tests is given below
+
++--------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Security Countermeasure        | Test                                                                                                                                                                 |
++================================+======================================================================================================================================================================+
+| BUS.INTEGRITY                  | ``riscv_mem_intg_error_test`` in Ibex DV.                                                                                                                            |
+|                                | The ``chip_sw_data_integrity`` OpenTitan top-level test will trigger integrity errors within the OpenTitan specific ``rv_core_ibex`` wrapper.                        |
+|                                | The TL-UL host adapter used in the OpenTitan specific ``rv_core_ibex`` is fully verified elsewhere in OpenTitan.                                                     |
++--------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| SCRAMBLE.KEY.SIDELOAD          | ``riscv_rand_instr_test`` in Ibex DV.                                                                                                                                |
+|                                | This test executes ``FENCE.I`` which rotates the scramble key                                                                                                        |
```

Comment:
Suggest to mention the recently added `rv_core_ibex_icache_invalidate_test` OT TLT (consistent with `BUS.INTEGRITY`) along the following lines:
> The `rv_core_ibex_icache_invalidate_test` OpenTitan top-level test covers assertions within the OpenTitan specific `rv_core_ibex` wrapper that check that a `FENCE.I` results in an icache scramble key request and that the returned key is correctly supplied to the scrambling memory primitives.